packetbeat/beater/packetbeat.go

```diff
@@ -49,11 +49,6 @@ type CmdLineArgs struct {
 
 var cmdLineArgs CmdLineArgs
 
-const (
-	defaultQueueSize     = 2048
```

Comment:
For packetbeat this is kind of a BC break but as it is only the internal default and we don't recommend to change this value anyway, worth mentioning?